src/components/FeaturesDemo/WebBrowser/ReportPage/Label.tsx

```diff
@@ -0,0 +1,19 @@
+import clsx from "clsx";
+import React from "react";
+
+interface LabelProps {
+    title: string;
+    value: string;
+    className?: string;
+}
+
+export function Label(props: LabelProps) {
+    return (
+        <div
+            className={clsx("flex flex-nowrap text-nowrap text-xs text-gray-500", props.className)}
+        >
+            <div className="inline px-1 py-0.5 font-semibold">{props.title}</div>
```

Comment:
if you need `inline` here, use `span` for it instead of `div`

Reply:
idk, why rely on default browser styling if we can use divs everywhere and style them as we want. but alright, replaced with spans